source/check-in.rst

```diff
@@ -41,20 +55,221 @@ Check-in credentials to obtain:
 * a client secret
 * a refresh token
 
-All of them can be used to obtain the needed access token, here using curl:
+Note them down so you can use them for the next steps.
+
+Discovering projects in Keystone
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Your access token will provide you access to a cloud provider, but you may have
+access to several different projects within that provider (a project can be
+considered as equal to a VO allocation). In order to discover which projects
+are available you can do that using the Keystone API.
+
+The :download:`../get-projects.py` script will just do that for you (requires
+``requests`` library). It expects these variables to be defined in the
+environment:
+
+* ``CHECKIN_CLIENT_ID``: Your Check-in client id (get it from https://aai.egi.eu/fedcloud)
```

Comment:
May want to use properly declared links.

Reply:
Fixed